roles/kubernetes-apps/helm/tasks/main.yml

```diff
@@ -27,13 +27,9 @@
   when: dns_mode != 'none' and inventory_hostname == groups['kube-master'][0] and rbac_enabled
 
 - name: Helm | Install/upgrade helm
-  command: "{{ bin_dir }}/helm init --upgrade --tiller-image={{ tiller_image_repo }}:{{ tiller_image_tag }}"
+  command: "{{ bin_dir }}/helm init --upgrade --tiller-image={{ tiller_image_repo }}:{{ tiller_image_tag }}{% if rbac_enabled %} --serviceaccount={{ system_namespace }}:tiller{% endif %}{% if tiller_node_selectors is defined%} --node-selectors {{ tiller_node_selectors }}{% endif %}{% if tiller_override %} --override {{ tiller_override }}{% endif %}"
```

Comment:
Can you add commented out options to roles/kubernetes-apps/helm/defaults/main.yml to show that these two vars exist? Just something basic like:

```
# Set node selector for tiller
#tiller_node_selectors: 

# Set override options
#tiller_override: "key1=val1,key2=val2"
```

Then update this line here:
```
  command: "{{ bin_dir }}/helm init --upgrade --tiller-image={{ tiller_image_repo }}:{{ tiller_image_tag }}{% if rbac_enabled %} --serviceaccount={{ system_namespace }}:tiller{% endif %}{% if tiller_node_selectors is defined%} --node-selectors {{ tiller_node_selectors }}{% endif %}{% if tiller_override is defined %} --override {{ tiller_override }}{% endif %}" 
```
( I just added "is defined")

Reply:
Missed that. thanks